jekyll/_cci2/server/v4.3/installation/installation-reference.adoc

```diff
@@ -1271,78 +1285,6 @@ a|Session Cookie Key. NOTE: Must be exactly 16 bytes.
 |`"transit"`
 |When `internal: true`, this value is used for the vault transit path.
 
-|`vm_gc.replicas`
-|int
-|`1`
-|Number of replicas to deploy for the vm-gc deployment.
-
-|`vm_scaler.prescaled`
-|list
-|`[{"count":0,"cron":"","docker-engine":true,"image":"docker-default","type":"l1.medium"},{"count":0,"cron":"","docker-engine":false,"image":"default","type":"l1.medium"},{"count":0,"cron":"","docker-engine":false,"image":"docker","type":"l1.large"},{"count":0,"cron":"","docker-engine":false,"image":"windows-default","type":"windows.medium"}]`
-|Configuration options for, and numbers of, prescaled instances.
-
-|`vm_scaler.replicas`
-|int
-|`1`
-|Number of replicas to deploy for the vm-scaler deployment.
-
-|`vm_service.dlc_lifespan_days`
```

Comment:
going to update values in readme and have them moved over later with other changes